settings.gradle

```diff
@@ -5,6 +5,7 @@ include 'examples'
 // Disable for now
 include 'ml'
 include 'assembly'
+include 'h2o-assembly'
```

Comment:
keep the same prefix please: `assembly-h2o`. It is more handy since if i am looking for that i can type assembly and shell will offer me alternatives. Think about it as hierarchical space...